flow/storage/elasticsearch/elasticsearch.go

```diff
@@ -135,7 +135,7 @@ type embeddedFlow struct {
 	Transport    *flow.TransportLayer `json:"Transport,omitempty"`
 	ICMP         *flow.ICMPLayer      `json:"ICMP,omitempty"`
 	DHCPv4       *fl.DHCPv4           `json:"DHCPv4,omitempty"`
-	DNS          *fl.DNS              `json:"DNS,omitempty"`
+	DNS          []*fl.DNS            `json:"DNS,omitempty"`
```

Comment:
I assume it is to keep query/reply ? I would prefer to keep on entry that we update with Answers/Questions. A more general remark. If we had too much thing I would not see the point to not use RawPackets

Reply:
done,
although I didn't really understand your remark regarding RawPackets and how you think we should use them, so if you could elaborate a bit further that would be really nice of you;)